### PR TITLE
Feat:Image and Typography

### PR DESCRIPTION
**Pull request by ArpitPatni**
Hey guys
Image and Typography component added.
Please review

### CODE REVIEW

File: Styles/Typography.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.Typo-items{
+    padding:20px;
+    border: 0.5px solid rgba(128, 128, 128, 0.363);
```
Comment (shubhmehra) [inline Styles/Typography.css]: use color from root

File: Styles/image.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.img-fluid{
+    width: 100%;
+    height: auto;
+}
+.image-items{
+    padding:20px;
+    border: 0.5px solid rgba(128, 128, 128, 0.363);
```
Comment (shubhmehra) [inline Styles/image.css]: same use from root

File: documentation.html [hunk 3]
```diff
@@ -356,6 +366,67 @@ <h3 class="avatar-heading">Floating action Button</h3>
...
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+
+      <h3 class="avatar-heading">Rounded Images</h3>
+      <p class="component-navbar-item">
+        You can add class img-round to make your image round shaped.
+      </p>
+
+      <div class="image-items">
+        <img class="img-round" src="Asset/image-3.jpg" alt="">
```
Comment (shubhmehra) [inline documentation.html]: use valid alt text

**Review by shubhmehra — APPROVED** [review]
add valid alt to images overall looks good to me

**Comment by yashodaAgarwal** [comment]
Looks good to merge

File: Styles/image.css (hunk 2 above)
Comment (ArpitPatni) [reply Styles/image.css]: Thank you made the following changes.

File: documentation.html (hunk 3 above)
Comment (ArpitPatni) [reply documentation.html]: Thank you made the following changes.

**Pull request merged**